components/prism-sas.js

```diff
@@ -237,7 +237,22 @@
 			lookbehind: true,
 			inside: args
 		},
-
+		'macro-string-functions': {
+			pattern: /((?:^|\s|=))%(?:NRBQUOTE|NRQUOTE|NRSTR|BQUOTE|QUOTE|STR)\(.*?(?:[^%]\))/i,
+			lookbehind: true,
+			inside: {
+				function: {
```

Comment:
Please wrap this and `string` and `punctuation` into single quotes to keep things consistent.

Reply:
Addressed as well as quoting function.